.changeset/curly-cows-buy.md

```diff
@@ -0,0 +1,6 @@
+---
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Alignment with Figma characteristics
```

Comment:
Could we go into a bit more detail here about the the changes please

Reply:
done